hcl/scanner/scanner.go

```diff
@@ -224,7 +224,11 @@ func (s *Scanner) scanComment(ch rune) {
 	// single line comments
 	if ch == '#' || (ch == '/' && s.peek() != '*') {
 		ch = s.next()
-		for ch != '\n' && ch >= 0 && ch != eof {
+		for ch != '\n' && ch != '\r' && ch >= 0 && ch != eof {
+			if ch == '\r' && s.peek() == '\n' {
```

Comment:
In other words, I believe this will break comments immediately on `\r` instead of only on `\r\n`... which might be fine.